utils/hct/hctgettaef.py

```diff
@@ -8,10 +8,16 @@
 taef_dir = os.path.join(src_dir, "external", "taef")
 
 if not os.path.isdir(taef_dir):
-  os.mkdir(taef_dir)
+  os.makedirs(taef_dir)
+
+try:
+  urllib.urlretrieve(url, zipfile_name)
+except:
+  print("Unable to read file with urllib, trying via powershell...")
+  from subprocess import check_call, check_output
```

Comment:
Could also just write the qualified version `subprocess.check_call`. Then it is easy to switch as well.